websites/M/Mediafire/presence.ts

```diff
@@ -2,151 +2,128 @@ const presence = new Presence({
+			else if (pathname.includes("/upgrade/"))
+				presenceData.details = "Upgrade Plans";
+			else if (pathname.includes("myaccount")) {
+				presenceData.details = `${document
+					.querySelector("[class~='current']")
+					.textContent.trim()} Settings`;
+			}
 
-		if (page.includes("/file/")) {
-			title = document.querySelector("div.dl-btn-label");
-			presenceData.details = "Viewing:";
-			presenceData.state = title.textContent;
-		}
-		if (page === "/software/mobile/" || page === "/software/mobile") {
-			presenceData.details = "Viewing:";
-			presenceData.state = "Mediafire Mobile Apps";
-		}
-		if (page === "/about" || page === "/about/") {
-			title = document.querySelector("#titlebar > div > h1");
-			presenceData.details = "Viewing:";
-			presenceData.state = title.textContent;
-		} else if (page.includes("/about/contact_us")) {
-			title = document.querySelector("#content > h2");
-			presenceData.details = "Viewing:";
-			presenceData.state = title.textContent;
-		} else if (page.includes("/about/jobs")) {
-			title = document.querySelector("#jobs_pics_section > div.opener > h2");
-			presenceData.details = "Viewing:";
-			presenceData.state = title.textContent;
-		}
-		if (page === "/advertising" || page === "/advertising/") {
-			title = document.querySelector("#content > h2");
-			presenceData.details = "Viewing:";
-			presenceData.state = title.textContent;
-		} else if (page.includes("/advertising/")) {
-			title = document.querySelector("#content > h2");
-			if (!title) {
-				title = document.querySelector("#adInquiryWrap > h2");
-				presenceData.details = "Viewing:";
-				presenceData.state = title.textContent;
-			} else {
-				presenceData.details = "Viewing:";
-				presenceData.state = title.textContent;
+			if (pathname !== "/") {
+				presenceData.buttons = [
+					{
+						label: "Open Mediafire",
+						url: href,
+					},
+				];
 			}
-		}
-		if (page === "/press" || page === "/press/") {
-			title = document.querySelector("#content > h2.h2.mb-4");
-			presenceData.details = "Viewing:";
-			presenceData.state = title.textContent;
-		} else if (page.includes("/press/inquiries")) {
-			title = document.querySelector("#content > h2");
-			presenceData.details = "Viewing:";
-			presenceData.state = title.textContent;
-		}
-		if (page.includes("/policies/")) {
-			title = document.querySelector("#content > div > div > h1 > strong");
-			if (!title) {
-				title = document.querySelector("#content > h2");
-				presenceData.details = "Viewing:";
-				presenceData.state = title.textContent;
+			break;
+		case "app.mediafire.com":
+			if (pathname.includes("/myfiles")) presenceData.details = "My Files";
+			else if (pathname.includes("/recent"))
+				presenceData.details = "My Recent Files";
+			else if (pathname.includes("/following"))
+				presenceData.details = "Files Shared With Me";
+			else if (pathname.includes("/trash"))
+				presenceData.details = "The Trashcan";
+			else presenceData.details = "My Files";
```

Comment:
All the strings here sound weird standalone. Add a verb before it - "Viewing their files" etc